spectree/config.py

```diff
@@ -63,3 +68,6 @@ def update(self, **kwargs):
 
         assert self.UI in self._SUPPORT_UI, "unsupported UI"
         assert self.MODE in self._SUPPORT_MODE, "unsupported MODE"
+        if self.AUTH_METHODS:
+            for value in self.AUTH_METHODS.values():
+                SecuritySchemesData(**value)
```

Comment:
Will it be better to accept the instance of `SecuritySchemesData`? So when the user creates this, the data is already validated. Then we can add another method to get the JSON schema for OpenAPI to use.

Reply:
Also, I'm not sure about `self.AUTH_METHODS` naming, maybe name from the OpenAPI scheme will be better?
For example - `self.securitySchemes`, not a pythonic style but... it is easy to understand what the parameter it is.

Reply:
Agree. `self.SECURITY_SCHEMES` sounds better. BTW, I'm thinking if we should use `pydantic.BaseSettings` for the config class.

Reply:
> pydantic.BaseSettings

Sounds good but for some new PR.